Doc/howto/isolating-extensions.rst

```diff
@@ -64,7 +64,7 @@ Enter Per-Module State
 
 Instead of focusing on per-interpreter state, Python's C API is evolving
 to better support the more granular *per-module* state.
-This means that C-level data is be attached to a *module object*.
+This means that C-level data can be attached to a *module object*.
```

Comment:
I believe this is a description of the status quo rather than a possibility:
```suggestion
This means that C-level data should be attached to a *module object*.
```

should be / will be / is / etc.

Reply:
Thanks for the suggestion. Based on some of the discussions I was reading in the old bpo, I agree this is better.